src/seedu/addressbook/data/person/Person.java

```diff
@@ -84,4 +84,16 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    public String getPrintableString(Printable... printables){
+
+        String output = "";
+
+        for(Printable people: printables){
+
+            output += people.getPrintableString() + "\n";
+        }
+
+        return output;
```

Comment:
- Unnecessary empty lines.
- Java has a `StringJoiner` class that can be used to join strings.